Sources/TokamakCore/Fiber/Layout/StackLayoutComputer.swift

```diff
@@ -0,0 +1,148 @@
+// Copyright 2021 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 5/24/22.
+//
+
+import Foundation
+
+final class StackLayoutComputer: LayoutComputer {
+  let proposedSize: CGSize
+  let axis: Axis
+  let alignment: Alignment
+  let spacing: CGFloat
+  /// A multiplier of `1` for the axis that takes the size of the largest child.
```

Comment:
Could you clarify the comment so that it's clear what "a multiplier of `1`" means in this context?

Reply:
Let me know if the new doc comment clears it up at all